<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUsername</class>
 <widget class="QDialog" name="DialogUsername">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create your account</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="layoutName">
       <item>
        <widget class="QLabel" name="labelName">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Your name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineName">
         <property name="maxLength">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutTag">
       <item>
        <widget class="QLabel" name="labelTag">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Account tag:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameTag">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="maxLength">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutPhone">
       <item>
        <widget class="QLabel" name="labelPhone">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Phone number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="phonenumCountry">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>UA (+380)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPhone">
         <property name="inputMask">
          <string>00-000-0000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutPass">
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Your password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linePassword">
         <property name="maxLength">
          <number>64</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSeePass">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../images.qrc">
           <normaloff>:/start/ui/images/blind.ico</normaloff>:/start/ui/images/blind.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
